starter-code/app.js

```diff
@@ -11,7 +11,7 @@ const path         = require('path');
 
 
 mongoose
-  .connect('mongodb://localhost/starter-code', {useNewUrlParser: true})
+  .connect('mongodb://localhost/mongoose-movies', { useNewUrlParser: true})
   .then(x => {
     console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`)
```

Comment:
Here you are repeating the code a little bit. You could have deleted this code and just required your db config file. It would do the same thing :)